### ISSUE TITLE
default max versions

### ISSUE THREAD

**Issue by webgeist** [wbolster/happybase#82]
Since version 0.96 default number of versions is 1. But in happybase this constant is 3 https://github.com/wbolster/happybase/blob/master/happybase/Hbase.thrift#L71. 
This difference in behavior is seems to be strange. Maybe should remove this from happybase thus the default number of versions will be determinated only in hbase?

**Comment by wbolster** [comment]
Right, I guess the default has changed upstream, since this file is just a copy.

Fwiw, I don't want local modifications to external files since that's a pain to maintain. Updating to a new version and regenerating the Thrift code is the way to go imho.
